projects/fsastorefrontlib/src/cms-components/user-profile/user-profile.component.html

```diff
@@ -78,8 +81,56 @@ <h6 class="notice">{{ user.name }}</h6>
                   }}
                 </span>
               </li>
+              <hr />
+              <li class="d-flex justify-content-between">
+                <span class="semi-bold pr-2"
+                  >{{ 'fscommon.register.address.label' | cxTranslate }}:</span
+                >
+                <div
+                  class="notice"
+                  *ngIf="user.defaultAddress; else addAddress"
+                >
+                  <div>
+                    {{ user.defaultAddress.country.name }}
+                  </div>
+                  <div>
+                    <span>
+                      {{ user.defaultAddress.line1 }}
+                    </span>
+                    <span>
+                      {{ user.defaultAddress.line2 }}
+                    </span>
+                  </div>
+                  <div>
+                    <span>
+                      {{ user.defaultAddress.town }}
+                    </span>
+                    <span>
+                      {{ user.defaultAddress.postalCode }}
+                    </span>
+                  </div>
+                  <div class="text-right mt-3">
+                    <a class="link" (click)="showUserAddressForm()">
```

Comment:
These anchors should perhaps be `span`s since they don't have any routing on them. The "link" class will take care of the styling.